### PR TITLE
piwik 3 compatibility

### PR DESCRIPTION
**Pull request by wronan**

### CODE REVIEW

File: Reports/GetPlatforms.php [hunk 1]
```diff
@@ -20,13 +21,14 @@ protected function init()
     {
         parent::init();
 
-        $this->category = 'General_Visitors';
-        $this->dimension = new Platform();
+        $this->categoryId = 'General_Visitors';
+        $this->subcategoryId = 'PlatformsReport_Platforms';
         $this->name = Piwik::translate('PlatformsReport_Platforms');
-        $this->documentation = Piwik::translate('PlatformsReport_GetPlatformsDocumentation');
         $this->widgetTitle = 'PlatformsReport_Platforms';
```
Comment (azarecki) [inline Reports/GetPlatforms.php]: Please remove widgetTitle and menuTitle, it is not used anymore

File: Reports/GetPlatformsWithVersions.php [hunk 2]
```diff
@@ -19,12 +21,12 @@ protected function init()
     {
         parent::init();
 
-        $this->category = 'General_Visitors';
-        $this->dimension = new Platform();
+        $this->categoryId = 'General_Visitors';
         $this->name = Piwik::translate('PlatformsReport_PlatformsWithVersions');
-        $this->documentation = Piwik::translate('PlatformsReport_GetPlatformsWithVersionsDocumentation');
         $this->widgetTitle = 'PlatformsReport_PlatformsWithVersions';
```
Comment (azarecki) [inline Reports/GetPlatformsWithVersions.php]: Please remove widgetTitle and add configureWidgets method for this report to be visible in dashboard

Comment (wronan) [inline Reports/GetPlatformsWithVersions.php]: I did that on purpose. It looks like this report should be available only through 'Related reports' section.

Comment (wronan) [inline Reports/GetPlatformsWithVersions.php]: Correction: there should be one menu entry, but both reports should be widgetizable on the dashboard separately.

**Comment by wronan** [comment]
OK @azarecki - please take another look when you are available.

**Pull request merged**